docs/Operators.md

```diff
@@ -7630,39 +7826,79 @@ This version of the operator has been available since version 16 of the default
 #### Attributes
 
 <dl>
-<dt><tt>align_corners</tt> : int (default is 0)</dt>
-<dd>If align_corners=1, the extrema (-1 and 1) are considered as referring to the center points of the input's corner pixels. If align_corners=0, they are instead considered as referring to the corner points of the input's corner pixels, making the sampling more resolution agnostic.</dd>
+<dt><tt>align_corners</tt> : int32 (default is 0)</dt>
```

Comment:
It should be `int` in Attributes.

Reply:
This is one of the issues pointed out in the doc. How many bits and signedness is "int"?